lib/brain.js

```diff
@@ -1262,7 +1281,10 @@ Brain.prototype.smsCompliance = function smsCompliance (opts = {}) {
     this.flow = null
     this.failedCompliance = 'sms'
 
-    if (returnState) return this.smsFlowHandleReturnState()
+    if (this.returnState && !_.includes(this.complianceReason, ARE_YOU_SURE_HANDLED_SMS_COMPLIANCE)) {
```

Comment:
I'm not sure if using the local returnReturn state was coding style or if it is to stop another method to change the global one, I could add another global variable instead of messing with this.